include/ck/tensor_operation/gpu/thread/threadwise_welford.hpp

```diff
@@ -75,4 +75,55 @@ struct ThreadwiseWelford
     int max_count_;
 };
 
+template <typename T, typename SrcMeanVarCountThreadDesc_M_K, typename DstMeanVarThreadDesc_M>
+struct ThreadwiseWelford_2
```

Comment:
Is it better to rename ThreadwiseWelford_2 to ThreadwiseWelfordMerge?

Reply:
See [8d29a](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/403/commits/f18361e8c109a485703a1820f3bb033487b8d29a)